### PR TITLE
corerouter: add lldpd

### PR DESCRIPTION
**Pull request by PolynomialDivision**
We need to get independent of the olsrd name protocol. Add lldpd as
standatarized protocol to get information about neighbors. To test it
just do

  lldpcli show neighbors

We can later use the information to feed owm with it.

### CODE REVIEW

File: roles/cfg_openwrt/templates/corerouter/config/lldpd.j2 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#jinja2: trim_blocks: "true", lstrip_blocks: "true"
+config lldpd config
+	option enable_cdp 0
+	option enable_fdp 0
+	option enable_sonmp 0
+	option enable_edp 0
+	option lldp_class 4
+	option lldp_location "2:FR:6:Commercial Rd:3:Roseville:19:4"
```
Comment (Noki) [inline roles/cfg_openwrt/templates/corerouter/config/lldpd.j2:8]: This should be properly filled with the location of the device.

Comment (PolynomialDivision) [reply roles/cfg_openwrt/templates/corerouter/config/lldpd.j2:8]: I think we can just delete this line.

**Review by Noki — APPROVED** [review]
I tested this and the data and output-formats look really good. I think it is a great idea to roll this out over all bbb-configs corerouter nodes so we can play around with the data. My only concern is that the package is quite big (131 KB).

**Comment by Akira25** [comment]
Before applying this, we should definitely discuss this in a meeting!

**Comment by spolack** [comment]
I'd also opt for not merging this at the moment. But we definitely should get documented the findings and pro/con arguments for using this over http endpoint.

**Pull request closed without merging**